pkg/security/apis/security/v1/types.go

```diff
@@ -81,6 +81,9 @@ type SecurityContextConstraints struct {
 	// used to generate a value for a pod the first non-wildcard profile will be used as
 	// the default.
 	SeccompProfiles []string `json:"seccompProfiles,omitempty" protobuf:"bytes,20,opt,name=seccompProfiles"`
+	// AllowedFlexDrivers is a whitelist of allowed Flexvolume drivers.
+	// Empty or nil indicates that all drivers may be used.
+	AllowedFlexDrivers []string `json:"allowedFlexDrivers" protobuf:"bytes,21,opt,name=allowedFlexDrivers"`
```

Comment:
> So this field is used when the user already has flexVolume power right? If they don't, then this is never used?

Correct.

> If that's the case, I'd expect a validation rule and some more detail in the doc.

At present, it's possible to have non-empty `AllowedFlexDrivers` and prohibited the usage of flexVolumes at the same time. Should we add validation to that case, and force user to have an empty `AllowedFlexDrivers`?

> Also, why not move up underneath Volumes? The proto number can still be 21 up there.

Hm. It's not clear what do you mean, so I put `AllowedFlexDrivers` field right after `Volumes` in the `pkg/security/apis/security/types.go` file. I left the order the same in `pkg/security/apis/security/v1/types.go` file, let me know if you want to see it updated too.